### ISSUE TITLE
Appending class

### ISSUE THREAD

**Issue by thipages** [WebReflection/lighterhtml#13]
Hi,
the following code raises an error which I understand.
However I would like to be able to have for class (or style) a static and a dynamic part.
I was thinking of managing it through the augmentor.
Any other simplier possibilities?

```javascript
import {render, html} from '//unpkg.com/lighterhtml?module';
const div=(classToAppend)=>html`
<div class="staticPart ${classToAppend}">text</div>
`;
// Error : index.js:875 Uncaught Error: 1 values instead of 0 <div class="staticPart , ">text</div>
render (document.body,()=>div("classToAppend"));
```

**Comment by thipages** [comment]
sorry I was quick on writing the code.
Updated.
Question remained

**Comment by WebReflection** [comment]
https://viperhtml.js.org/hyperhtml/documentation/#essentials-7

**Comment by WebReflection** [comment]
> I would like to be able to have for class (or style) a static and a dynamic part.

It's impossible, due DOM nature, but you can always have what you are looking for:

```js
import {render, html} from '//unpkg.com/lighterhtml?module';
const div=(classToAppend)=>html`
<div class="${['staticPart'].concat(classToAppend).join(' ')}">text</div>
`;
```

**Comment by WebReflection** [comment]
TL;DR attributes don't **ever** change partially, so that partial attributes are both logically wrong and, practically, a lie.

**Comment by thipages** [comment]
very clear now and important. Thank you
